src/sentry/sentry_metrics/aggregation_option_registry.py

```diff
@@ -9,20 +9,27 @@ class AggregationOption(Enum):
     TEN_SECOND = "ten_second"
 
 
+class TimeWindow(Enum):
+    SEVEN_DAYS = "7d"
+    FOURTEEN_DAYS = "14d"
+    THIRTY_DAYS = "30d"
+    NINETY_DAYS = "90d"
+
+
 METRIC_ID_AGG_OPTION = {
-    "d:transactions/measurements.fcp@millisecond": AggregationOption.HIST,
-    "d:transactions/measurements.lcp@millisecond": AggregationOption.HIST,
+    "d:transactions/measurements.fcp@millisecond": {AggregationOption.HIST: TimeWindow.NINETY_DAYS},
+    "d:transactions/measurements.lcp@millisecond": {AggregationOption.HIST: TimeWindow.NINETY_DAYS},
 }
 
-USE_CASE_AGG_OPTION = {UseCaseID.CUSTOM: AggregationOption.TEN_SECOND}
+USE_CASE_AGG_OPTION = {UseCaseID.CUSTOM: {AggregationOption.TEN_SECOND: TimeWindow.SEVEN_DAYS}}
```

Comment:
By extending the fields to now map to a dictionary, do we think we'll need support for multiple aggregations per MRI or per use case? Today if I understand correctly we only support 1 aggregation option override per MRI/use case.